reference/audit_logs.rst

```diff
@@ -0,0 +1,241 @@
+JAAS: Understanding Audit Logs
+==============================
+
+Introduction
+------------
+
+JIMM provides audit logging functionality, tracking all requests/responses into the system.
+This gives administrators of JIMM the ability to audit changes at a very granular level.
+
+All requests to controllers and models are logged and can enable an analysis into why the state
+of the underyling Juju estate has changed.
+
+Audit logs are currently enabled by default without any config to disable their collection.
+The logging retention period is configurable and allows for audit logs to be stored for a
+variable amount of time depending on your requirements and storage constraints.
+
+Filtering audit logs is also possible along a variety of fields including but not limited to the
+user executing the request, model name and command.
+
+Finally, access to audit logs is, by default, only afforded to JIMM superuers i.e. administrators of JIMM.
+Read access to audit logs can be granted to other users via jimmctl, see :doc:`/tutorial/group_management`
+
+Filter Logs
+-----------
+Querying for audit logs is most readily done via the jimmctl CLI tool.
+
+Basics
+~~~~~~
+
+``jimmctl audit-events --help``
+
+will show you a list of all available filter options
+
+``jimmctl audit-events``
+
+will return a list of audit logs. This can be filtered and paginated through as described below.
+
+Each audit log contains the following pieces of information:
+
+- **time**:             When the audit log was created.
+- **conversation-id**:  A unique id per websocket connection.
+- **message-id**:       An incrementing count for each message sent during a session
+- **facade-name**:      The name of the grouping of methods a call is intended for, a facade categorises methods.
+- **facade-method**:    The name of the method called
+- **facade-version**:   The version of the facade called, different client versions may use different facade version.
+- **object-id**:        A parameter used in some methods, indicates the object being acted upon.
+- **user-tag**:         The user making the request.
+- **model**:            The model a request is acting on, can be empty for controller level requests.
+- **is-response**:      Indicates whether this log is for a request or response message.
```

Comment:
Let's put the types here, like **is-reponse[boolean]**

Reply:
Good idea